app/main/views/search.py

```diff
@@ -110,7 +110,19 @@ def supplier_search():
                 }
             },
             "sort": sort_queries,
-        }
+            }
+
+    elif keyword:
+        query = {
+            "query": {
+                "wildcard": {
```

Comment:
used `match_phrase_prefix` because `match` did not return results for partial words.